lib.php

```diff
@@ -1333,3 +1334,14 @@ function bigbluebuttonbn_has_capability($bigbluebuttonbnid, $capability) {
     }
     return true;
 }
+
+
+/**
+ * Helper for generating access codes for BBB
+ *
+ * @return string
+ */
+function bigbluebuttonbn_generate_access_code() {
+    return rand(1, 999999);
```

Comment:
Does this have to be unique, or can there be collisions?

Reply:
It does not, since it's tied to each specific instance of the BBB session. If anything, the guestlinkid (different place) should be unique, but it is not currently.